debian/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM hexpm/elixir:1.15.7-erlang-26.1.1-debian-bookworm-20230612-slim
+
+ARG host_user_name
+ARG host_group_name
+ARG host_uid
+ARG host_gid
+ARG host_ostype
+
+RUN apt-get update -y && apt-get install -y \
+      build-essential \
+      git \
+      inotify-tools \
+      nodejs \
+      npm \
+    && apt-get clean && rm -f /var/lib/apt/lists/*_*
+
+RUN if [ "$host_ostype" = "Linux" ]; then \
```

Comment:
@mnishiguchi 
It depends on OS type, so I think it may not work well if someone use Linux and `Rootless Docker`.
However in that case, we change `host_ostype` in .env 👍